integration_tests/pages/goal/AddNotePage.ts

```diff
@@ -19,7 +19,7 @@ export default class AddNotePage extends Page {
     this.submitButton().click()
   }
 
-  noteField = (): PageElement => cy.get('#goal-note')
+  noteField = (): PageElement => cy.get('#notes')
```

Comment:
I note we're using a mix of `note` and `notes`. The latter is used in the API and I thought it makes sense to have the form field match it, but the singular is used on the screens itself.